esx_service/vsan_policy_test.py

```diff
@@ -62,6 +62,7 @@ def test_create_delete(self):
 
     def test_delete_nonexistent_policy_fails(self):
         self.assertNotEqual(None, vsan_policy.delete(self.name))
+        print ("The test itself is expected to fail, and please ignore the errors printed above.")
```

Comment:
does not it have to be stderr ? Or log ? (just asking  since I do not see this message in the log